heft-plugins/heft-storybook-plugin/src/StorybookPlugin.ts

```diff
@@ -117,14 +130,35 @@ export class StorybookPlugin implements IHeftPlugin<IStorybookPluginOptions> {
         '(EXPERIMENTAL) Used by the "@rushstack/heft-storybook-plugin" package to launch Storybook.'
     });
 
+    const storybookStaticBuildParameters: IHeftFlagParameter = heftSession.commandLine.registerFlagParameter({
```

Comment:
Is this separate from `storybookParameters` just because the documentation is different between the two? Seems like we could update the phrasing in the docs and deduplicate these.

Reply:
Yeah I like that approach better than having separate parameters

Reply:
Actually, are action names available in legacy Heft? If they aren't, then the only way to differentiate between the two scenarios would be the separate parameters.

Reply:
Do you know where to get the action name? I searched in the heftSession, but I did not find one about the actionName, so I added another parameter.

Reply:
You just need to add `'build'` to the `associatedActionNames` array in the other parameter, and rephrase the documentation.

Reply:
Yeah, but the behaviour is different depending on the action. How would you differentiate between which action was called?

Reply:
You can look at the `build.properties.serveMode` property to switch on behavior.

Reply:
updated